src/lib.rs

```diff
@@ -438,6 +438,26 @@ impl Pixels {
         &mut self.pixels
     }
 
+    /// Get a mutable u32 slice for the pixel buffer.  The buffer is _not_
+    /// cleared for you; it will retain the previous frame's contents until you
+    /// clear it yourself.
+    ///
+    /// This function will fail if the internal pixels array is not 4-byte
+    /// aligned, which should always be the case.
+    ///
+    /// The pixel channels order is 0xAABBGGRRu32.  For example, 0x12345678u32
+    /// will produce an alpha of 0x12, a blue of 0x34, a green of 0x56 and a red
+    /// of 0x78.
+    pub fn get_frame_u32(&mut self) -> &mut [u32] {
```

Comment:
I thought about using a Result but then I realised that it will probably never fail.  Memory allocations are always 4-byte aligned and I am sure your internal u8 slices are multiples of 4 bytes.  I didn't see a point of the caller requiring to handle a result when I was thinking there would never be an error.

Perhaps it would be better for me to shadow that with an `expect` call rather than the `if let`.

Also, perhaps the comment should be removed that describes the bit format if you do believe the texture format will not be the same.  How would you find out the texture format in that case?

Reply:
On the subject of endian awareness, if the texture format isn't constant on platforms, then we can just as well push the endianness issue to the user too.

Reply:
The texture format is configurable with [`PixelsBuilder::texture_format()`](https://docs.rs/pixels/0.6.0/pixels/struct.PixelsBuilder.html#method.texture_format). The default format is RGBA, which on a little endian machine will certainly be rendered as described in the comment; `0xAABBGGRR_u32`.